From 3e32f46c7948f6b26476558905995ebec9b01760 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Fri, 15 Oct 2021 23:36:54 +0200
Subject: [PATCH] cmake/modules/FindFFTW.cmake: fix build without fftw3

Build on Linux fails if libfftw3 is not available since commit
https://github.com/greatscottgadgets/hackrf/commit/a8c1fc92e9fa10843cb4aafd68c52f9903ebd05d
which replaced
pkg_check_modules(FFTW REQUIRED fftw3f)
by
find_package(FFTW REQUIRED)

Fix this build failure by updating FindFFTW.cmake to check for fftw3f

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Retrieved from:
https://github.com/greatscottgadgets/hackrf/commit/3e32f46c7948f6b26476558905995ebec9b01760]
---
 host/cmake/modules/FindFFTW.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/host/cmake/modules/FindFFTW.cmake b/host/cmake/modules/FindFFTW.cmake
index b9f3bfb81..802db4f5b 100644
--- a/host/cmake/modules/FindFFTW.cmake
+++ b/host/cmake/modules/FindFFTW.cmake
@@ -16,7 +16,7 @@ IF (WIN32)
 include_directories(${FFTW_INCLUDES})
 find_library (FFTW_LIBRARIES NAMES ${FFTW_LIBRARIES})
 ELSE(WIN32)
-find_library (FFTW_LIBRARIES NAMES fftw3)
+find_library (FFTW_LIBRARIES NAMES fftw3f)
 ENDIF(WIN32)
 
 
